### ISSUE TITLE
Create landing page for Conscious Parenting initiative

### ISSUE THREAD

**Issue by EilidhRoss1** [life-itself/community#308]
We want to have a landing page on the website for the Conscious Parenting project with Esther Montmany

## Acceptance

- [ ] Webpage is up

## Tasks

- [ ] Create webpage

## Copy

- Title
- Tagline
- Meta (start/end, team) Esther Montmany (link to her people page)
- Description
- Key Resources (links to articles via categoric sorting process) nb. if none hide title

**Conscious Parenting** with Esther Montmany
__Creating a wiser, weller world starts with great parenting. Join us in discovering conscious parenting to create the stewards of the future.__
Jan 2021 - Present 

**What**
Our conscious parenting project is a series of offerings from conscious parenting pioneer Esther Montmany. The offerings shed light on the impact of parents behaviours on the child, how to positively transform the role of the parents, and pragmatic tools for all elements of parenting. 

Esther has worked with Mauricio and Rebecca Wild at their legendary school in Ecuador and led the children's program at Plum Village summer retreats as well as starting several initiatives of her own. Over the past 20 years she has been exploring how to relate with children through her own experiences as a mother and as a coach for other parents. 

**Why** 
The next generation of humans are being born into and will live through some of the most turbulent times in human history. According to the NHS, [18% of children in the UK aged 7 to 16 have a probable mental disorder in 2022](https://digital.nhs.uk/data-and-information/publications/statistical/mental-health-of-children-and-young-people-in-england/2022-follow-up-to-the-2017-survey#:~:text=In%202022%2C%2018.0%25%20of%20children,between%202020%2C%202021%20and%202022.) and this number is steadily on the rise. Children need conscious parents more than ever, to help them navigate the modern world. 

Conscious parenting opens up the possibility of raising healthy children with deep awareness and groundedness who can go on to become brilliant adults. 

include links to podcasts, hangout and blog post.

**Comment by nathenf** [comment]
@rufuspollock please review the copy. I feel a lack of clarity about what this project actually is so it would be helpful if you had a SCQH (I can't find one) or we can discuss what the plan is. This would help me write better copy 😁

**Comment by nathenf** [comment]
@rufuspollock I propose that we use the same framework for the ecosystem listings as our projects now and then iterate

**Comment by rufuspollock** [comment]
@nathenf this and the original parent issue #156 are starting to be very duplicative e.g. the other issue had a whole section for landing page plus acceptance. Either we can converge that part here (and remove stuff from parent and just link to this as sub-issue) or merge back.

> @rufuspollock please review the copy. I feel a lack of clarity about what this project actually is so it would be helpful if you had a SCQH (I can't find one) or we can discuss what the plan is. This would help me write better copy 😁

What copy is great. I would leave out why copy for now. as stated in #156 i imagine this as a basic landing page for now with intro (your what) plus links to videos. Later we can add more. (again more on this in #156).

**Comment by rufuspollock** [comment]
@nathenf let's merge back. To clarify: let's create this once projects template is ready (or use this as a pilot for that if you wish since it is greenfield).

**Comment by nathenf** [comment]
WONT FIX - Merged into #156

**Comment by rufuspollock** [comment]
@nathenf added the `duplicate` label (this is duplicate not wontfix strictly).
